### PR TITLE
test interval coverage

### PR DESCRIPTION
**Pull request by starkari**
resolves issue #135

### CODE REVIEW

File: tests/testthat/test-score_forecasts.R [hunk 1]
```diff
@@ -2456,3 +2456,403 @@ test_that("wis is sum of components, 2 intervals, median and point, use_median_a
...
+    value = point_forecast,
+    stringsAsFactors = FALSE
+  )
+  actual <- score_forecasts(forecasts = test_forecasts, truth = test_truth, use_median_as_point = FALSE)
+  
+  expect_equal("coverage_0" %in% names(actual),FALSE)
+  
+})
+
+test_that("interval coverage is correct, median only, use_median_as_point TRUE", {
```
Comment (nickreich) [inline tests/testthat/test-score_forecasts.R:2508]: I think if there is only a median, no interval coverage should be returned

Comment (starkari) [reply tests/testthat/test-score_forecasts.R:2508]: There is a coverage 0 essentially saying if they accurately forecasted

File: tests/testthat/test-score_forecasts.R [hunk 2]
```diff
@@ -2456,3 +2456,403 @@ test_that("wis is sum of components, 2 intervals, median and point, use_median_a
...
+    type = rep("quantile", n_forecasts),
+    quantile = forecast_quantile_probs,
+    value = forecast_quantiles,
+    stringsAsFactors = FALSE
+  )
+  
+  actual <- score_forecasts(forecasts = test_forecasts, truth = test_truth, 
+                            use_median_as_point = TRUE)
+  
+  expected_coverage_0 <- ifelse(y==forecast_quantiles_matrix[, 3], 1, 0)
```
Comment (nickreich) [inline tests/testthat/test-score_forecasts.R:2675]: again, I think we should remove the coverage_0 as it doesn't make sense as an output.

**Review by nickreich — CHANGES_REQUESTED** [review]
This may require a change to score_forecasts() itself, but I think coverage_0 need not be returned and therefore does not need to be tested. otherwise, I reviewed the logic/code for the coverage tests and they seemed ok to me.

**Pull request merged**